code/backend/pages/01_Ingest_Data.py

```diff
@@ -145,7 +145,7 @@ def upload_file(bytes_data: bytes, file_name: str, content_type: Optional[str] =
             bytes_data,
             overwrite=True,
             content_settings=ContentSettings(content_type=content_type + charset),
-            metadata={"source_url": file_name},
+            metadata={"title": file_name},
```

Comment:
I'm nervous that this is effectively a breaking change (unless I have misunderstood)

Reply:
This should not be a breaking change. This was added in my last PR. Before that in push model this field is never used. I am updating it so that I am able to handle the '/' problem when pushing URL to blob storage. We can discuss more on this